data/registry/collector-receiver-openllmetry.yml

```diff
@@ -0,0 +1,19 @@
+# cSpell:ignore openllmetry Instana
+title: IBM OpenTelemetry Collector for LLM
```

Comment:
The repository title is "OTel based Data Collector for LLM", what is the difference here?

Also "IBM OpenTelemetry Collector for LLM" goes against the trademark requirements for OpenTelemetry, see https://www.linuxfoundation.org/legal/trademark-usage

Reply:
The title has been changed to be same as in repository.